internal/api/grpc/system/instance.go

```diff
@@ -95,6 +96,23 @@ func (s *Server) RemoveInstance(ctx context.Context, req *system_pb.RemoveInstan
 	}, nil
 }
 
+func (s *Server) ListIAMMembers(ctx context.Context, req *system_pb.ListIAMMembersRequest) (*system_pb.ListIAMMembersResponse, error) {
+	ctx = authz.WithInstanceID(ctx, req.InstanceId)
+	queries, err := ListIAMMembersRequestToQuery(req)
+	if err != nil {
+		return nil, err
+	}
+	res, err := s.query.IAMMembers(ctx, queries, false)
+	if err != nil {
+		return nil, err
+	}
+	return &system_pb.ListIAMMembersResponse{
+		Details: object.ToListDetails(res.Count, res.Sequence, res.Timestamp),
+		//TODO: resource owner of user of the member instead of the membership resource owner
+		Result: member.MembersToPb("", res.Members),
```

Comment:
it's copied knowingly from ListIAMMembers from the admin API, should they be handled differently, or is it just that there has to be created a new issue?

Reply:
ok i expected that admin API is correct. I'm fine with both: implementing it right now (incl. admin API) or creating a new issue. New issue might be even better, so that we have the basic functionally and only miss the avatar and implementation in the system API could be tricky as requestedHost is not available.

Reply:
created a userstory https://github.com/zitadel/zitadel/issues/5064